content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: OpenCensus Agent
+description: Learn how to configure the proxies to send OpenCensus-formatted spans to OpenTelemetry Collector.
+weight: 10
+keywords: [telemetry,tracing,opencensus,opentelemetry,span]
+aliases:
+    - /docs/tasks/opencensusagent-tracing.html
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+After completing this task, you will understand how to have your application participate in tracing with the OpenCensus Agent, export those traces to the OpenTelemetry collector, and have the OpenTelemetry collector export those spans to Jaeger.
+
+To learn how Istio handles tracing, visit this task's [overview](../overview).
+
+{{< boilerplate before-you-begin-egress >}}
+
+* Install [Jaeger](/docs/ops/integrations/jaeger/#installation) into your cluster.
+
+* Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
+
+## Configure tracing
+
+If you used an `IstioOperator` CR to install Istio, add the following field to your configuration:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+    meshConfig:
+        defaultProviders:
+            tracing:
+            - "opencensus"
+        enableTracing: true
+        extensionProviders:
+        - name: "opencensus"
+          opencensus:
+              service: "opentelemetry-collector.istio-system.svc.cluster.local"
+              port: 55678
+              context:
+              - W3C_TRACE_CONTEXT
+{{< /text >}}
+
+Otherwise, add the equivalent setting to your original `istioctl install` command, for example:
+
+{{< text syntax=bash snip_id=none >}}
+$ istioctl install -f <your-istio-operator-config-file>
```

Comment:
My confusion at this point is I don't see any new settings on what would be the original command.

Reply:
I remove this part